compressor/tests/test_offline.py

```diff
@@ -560,21 +562,10 @@ class OfflineCompressTestCaseErrors(OfflineTestCaseMixin, TestCase):
     templates_dir = 'test_error_handling'
 
     def _test_offline(self, engine):
-        count, result = CompressCommand().compress(
+        count, result = CompressCommand().handle_inner(
             log=self.log, verbosity=self.verbosity, engine=engine)
 
-        if engine == 'django':
-            self.assertEqual(2, count)
-        else:
-            # Because we use env.parse in Jinja2Parser, the engine does not
-            # actually load the 'extends' and 'includes' templates, and so
-            # it is unable to detect that they are missing. So all the
-            # 'compress' nodes are processed correctly.
-            self.assertEqual(4, count)
-            self.assertEqual(engine, 'jinja2')
-            self.assertIn(self._render_link('7ff52cb38987'), result)
-            self.assertIn(self._render_link('2db2b4d36380'), result)
-
+        self.assertEqual(2, count)
```

Comment:
Seems that this PR "fixes" a bug in compressor management of jira's templates, there's only 2 non bugged templates and compressor should return only 2 results for both engines.

But I wanted to look a bit more closely at this, and forgot to do so before pushing the PR, thanks for pointing this again to me :)